modules/nw-sriov-hugepages.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/about-sriov.adoc
+
+[id="nw-sriov-hugepages_{context}"]
+= Hugepages resource injection for Downward API
+
+When a pod specification includes a resource request or limit for hugepages, the SR-IOV network resource injector automatically adds fields to the pod specification to support consuming the hugepages information with the Downward API.
```

Comment:
Global comment for this file: "About the Network Resources Injector", uses "Network Resources Injector" everywhere. This file uses "the SR-IOV network resource injector" and "the resource injector".